src/lib/components/Row.js

```diff
@@ -155,14 +155,16 @@ export default class Row extends React.Component {
             nodeSize,
             nodeIcon,
         } = this.props;
-        const hasChildren = element.children && element.children.length > 0;
+        const showJunction = element.hasChildren
+        const hasChildren = element && element.children && element.children.length > 0
```

Comment:
Same as the comment in the README. I think the `hasChildren` attribute is useless; you could just check the existence of a `children` attribute